bigframes/core/nodes.py

```diff
@@ -341,15 +363,14 @@ def order_ambiguous(self) -> bool:
     def explicitly_ordered(self) -> bool:
         return True
 
-    def transform_children(
-        self, t: Callable[[BigFrameNode], BigFrameNode]
-    ) -> BigFrameNode:
-        return self
+    @property
+    def row_count(self) -> typing.Optional[int]:
+        return self.n_rows
 
 
 ## Put ordering in here or just add order_by node above?
 @dataclass(frozen=True)
-class ReadTableNode(BigFrameNode):
+class ReadTableNode(LeafNode):
```

Comment:
Do we want a `row_count` here? If there are no filters applied, we should be able to get this from table metadata (though it could be slightly inaccurate if there was anything in the streaming buffer, which can show up in query results before the count arrives at the table metadata).

Reply:
Yes, that is probably right. In new revision will stuff some more table metadata into these nodes and use that to provide row_count where possible.